views/trials-details.html

```diff
@@ -7,6 +7,15 @@
 {% endmacro %}
 
 {% block bannerFooter %}
+
+  <ul class="breadcrumb">
+    {% block breadcrumbs %}
+      <li><a href="/">Home</a></li>
+      <li><a>Trials</a></li>
+      <li><a>{{ title | truncate(35) }}</a></li>
+    {% endblock %}
+  </ul>
+
```

Comment:
I think you can avoid redefining the breadcrumb `ul` here by using `{{ super() }}` or, even better, getting the `breadcrumbs` block outside of the `bannerFooter` on the layout.